gcloud/orb.yaml

```diff
@@ -72,6 +72,7 @@ commands:
             equal: [ <<parameters.kubectl_version>>, "gcloud-latest" ]
           steps:
             - run: echo y | gcloud components install kubectl
+            - run: ln -s /root/google-cloud-sdk/bin/kubectl /usr/bin/kubectl
```

Comment:
We could also put this in `/usr/local/bin/kubectl` for consistency with the below cases.

Reply:
Probably makes sense to keep 'em consistent, yeah.